### PR TITLE
feat(crons): New onboarding panel with platform options

### PR DESCRIPTION
**Pull request by davidenwang**
This will be behind the `crons-new-onboarding` feature flag. Currently all the platform options simply lead to the create monitor page.

In the future each platform will replace the panel content with its own instrumentation quickstart guide

Before:
image

After:
image

Depends on: https://github.com/getsentry/sentry/pull/55410

### CODE REVIEW

File: static/app/views/monitors/components/platformPickerPanel.tsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+      </Actions>
+    </OnboardingPanel>
+  );
+}
+
+const OnboardingTitle = styled('h3')`
+  margin-bottom: ${space(1)};
+`;
+
+const SectionTitle = styled('div')`
```
Comment (evanpurkhiser) [inline static/app/views/monitors/components/platformPickerPanel.tsx:63]: Should we use a `hX` for this I wonder?

File: static/app/views/monitors/components/platformPickerPanel.tsx [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+          </PlatformOption>
+        ))}
+      </Actions>
+      <SectionTitle>{t('Generic')}</SectionTitle>
+      <Actions>
+        <NewMonitorButton size="sm" priority="default">
+          Sentry CLI
+        </NewMonitorButton>
+        <NewMonitorButton size="sm" priority="default">
+          HTTP (cURL)
```
Comment (evanpurkhiser) [inline static/app/views/monitors/components/platformPickerPanel.tsx:52]: 👍 no need for `t` here

File: static/app/views/monitors/components/platformPickerPanel.tsx [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+          "We'll tell you if your recurring jobs are running on schedule, failing, or succeeding."
+        )}
+      </p>
+      <SectionTitle>{t('Platforms')}</SectionTitle>
+      <Actions>
+        {CRON_SDK_PLATFORMS.map(({platform, label}) => (
+          <PlatformOption key={platform}>
+            <PlatformButton priority="default">
+              <PlatformIcon platform={platform} format="lg" size="100%" />
+            </PlatformButton>
```
Comment (evanpurkhiser) [inline static/app/views/monitors/components/platformPickerPanel.tsx:41]: Do we need some aria labels?

**Review by evanpurkhiser — APPROVED** [review]

File: static/app/views/monitors/components/platformPickerPanel.tsx (hunk 3 above)
Comment (davidenwang) [reply static/app/views/monitors/components/platformPickerPanel.tsx:41]: ahh yes can add!

**Pull request merged**